hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerStateManagerImpl.java

```diff
@@ -488,14 +511,19 @@ private ContainerInfo findContainerWithSpace(final long size,
 
   public void removeContainer(final HddsProtos.ContainerID id)
       throws IOException {
-    lock.writeLock().lock();
+    final ContainerID cid = ContainerID.getFromProtobuf(id);
+    lock.writeLock();
```

Comment:
Looks like a typo, lock needs to be acquired.

```suggestion
    lock.writeLock().lock();
```

Reply:
yes, I fixed and tested again. @adoroszlai , thanks for pointing out. Pls re-review.